apps/maps/widgets.py

```diff
@@ -21,14 +24,29 @@ class Media:
             staticfiles_storage.url('leaflet.draw.css'),
         ]}
 
+    def get_presets(self, category):
+        presets = MapPreset.objects.filter(category=category).all()
```

Comment:
I thought it was necessary to actually execute the query. If it is not, feel free to remove it.